[Docs-2.0] FAQ-UseWasabi small update 3 (#1098)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

:::details
### How can I change the observers of my receive address?

You can change the observers of your receive address, as long as it hasn't received any coins yet.

> This can be done through `Reveive->Unused Receive Addresses->Edit`, in the right side of the label.

Reviewer: ```suggestion
Go to the Unused Receive Addresses list:  `Receive->Unused Receive Addresses`, and click on the pencil icon to edit the labels.
```

> Go to the Unused Receive Addresses list:  `Receive->Unused Receive Addresses`, and click on the pencil icon to edit the labels.

Reviewer 2: ```suggestion
Go to the Unused Receive Addresses list:  `Receive` -> `Unused Receive Addresses`, and click on the pencil icon to edit the labels.
```

There is a risk that both of them send coins to this same destination, thus unnecessarily linking the payments.

:::danger
It is especially important to **NEVER** send your extended public key to any third party.
This is a complete de-anonymization of your entire wallet!!
:::
::::

:::details
### Why does Wasabi only use SegWit bech32 addresses?

> Wasabi was created after the activation of SegWit, and it made sense to support the most advanced address type, which has numerous benefits.

Reviewer 2: ```suggestion
Wasabi was created after the activation of SegWit, and it made sense to support the most advanced address type, which has numerous benefits, like large savings on mining fees for SegWit transactions.
```

There is a risk that both of them send coins to this same destination, thus unnecessarily linking the payments.

:::danger
It is especially important to **NEVER** send your extended public key to any third party.
This is a complete de-anonymization of your entire wallet!!
:::
::::

:::details
### Why does Wasabi only use SegWit bech32 addresses?

Wasabi was created after the activation of SegWit, and it made sense to support the most advanced address type, which has numerous benefits.

> For example, due to the malleability fix of SegWit, you can now remix coinjoin outputs which are currently unconfirmed.

Reviewer: ```suggestion
For example, due to the malleability fix of SegWit, you can now remix coinjoin outputs which are currently unconfirmed.
```
```suggestion
For example, due to the malleability fix of SegWit, you can now remix coinjoin outputs which are currently unconfirmed.
```
```suggestion
For example, due to the malleability fix of SegWit, you can now remix coinjoin outputs which are currently unconfirmed.
```

> For example, due to the malleability fix of SegWit, you can now remix coinjoin outputs which are currently unconfirmed.

Reviewer: in 2.0 only confirmed coins can coinjoin

:::

:::details
### Where can I find my address and the corresponding QR code?

You can see the address and its QR code when you create it through the `Receive` dialog.
To save the png file of the QR code, hover your mouse over the QR code and click `Save` button that appears to the right side.

![Wasabi Wallet address' QR code](/ReceiveAddress.png "Wasabi Wallet address' QR code")

You can also see your unused receive addresses and their QR codes.

![Wasabi Wallet address' QR code](/UnusedReceiveAddresses.png "Wasabi Wallet address' QR code")

Currently, there's no way to see used addresses and their QR codes through the GUI.
:::

:::details
### What derivation paths does Wasabi use?

Wasabi follows [BIP 84: Derivation scheme for P2WPKH Based Accounts](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki), so the main path is `m/84'/0'/0'`.

> On TestNet and RegTest Wasabi uses `m/84'/0'/0'`, and not the standard `m/84'/1'/0'`.

Reviewer 2: ```suggestion
On TestNet Wasabi uses the path `m/84'/1'/0'`.
On RegTest it uses `m/84'/0'/0'`, and not the standard `m/84'/1'/0'` path.
```

:::

:::details
### Where can I find my address and the corresponding QR code?

You can see the address and its QR code when you create it through the `Receive` dialog.
To save the png file of the QR code, hover your mouse over the QR code and click `Save` button that appears to the right side.

> ![Wasabi Wallet address' QR code](/ReceiveAddress.png "Wasabi Wallet address' QR code")

Reviewer 2: There is no such a file `ReceiveAddress.png`

Author: True. Its in https://github.com/zkSNACKs/WasabiDoc/pull/1088/files

:::

:::details
### Where can I find my address and the corresponding QR code?

> You can see the address and its QR code when you create it through `Receive` tab.

Reviewer: ```suggestion
You can see the address and its QR code when you create it through the `Receive` dialog.
```
